apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/open-patterns-panel.js

```diff
@@ -0,0 +1,41 @@
+// Simulate click that react responds to
+const mouseClickEvents = [ 'mousedown', 'click', 'mouseup' ];
+const simulateMouseClick = function ( element ) {
+	mouseClickEvents.forEach( ( mouseEventType ) =>
+		element.dispatchEvent(
+			new window.MouseEvent( mouseEventType, {
+				view: window,
+				bubbles: true,
+				cancelable: true,
+				buttons: 1,
+			} )
+		)
+	);
+};
```

Comment:
Note: If you're attempting to add breakpoints to the click simulation codes, bear in mind that React is sensitive to the timing between the chain of events between mousedown/click/mouseup. The inserter panel might open, but somehow, the button's click handler won't work anymore after that. At least that was my experience.